docs/source/models.rst

```diff
@@ -194,6 +194,11 @@ TensorFlow (``tensorflow``)
 
 The TensorFlow model flavor enables logging TensorFlow ``Saved Models`` and loading them back as ``Python Function`` models for inference on Pandas DataFrames. Given a directory containing a saved model, you can log the model to MLflow via ``log_saved_model``. The saved model can then be loaded for inference via ``load_pyfunc()``. For more information, see :py:mod:`mlflow.tensorflow`. 
 
+H\ :sub:`2`\ O (``h2o``)
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+With the h2o model flavor h2o models can be handled by mlflow. These models will be saved by using the :py:mod:`h2o.save_model`. Using :py:mod:`mlflow.h2o.log_model` will also enable a valid ``Python Function`` flavor.
```

Comment:
micronit: h2o -> H2O? (seems the pattern is to use h2o in code but H2O when referring to the library in texzt)